runtime/onert/core/src/backend/controlflow/ExternalContext.h

```diff
@@ -19,7 +19,13 @@
 
 #include <backend/IExternalContext.h>
 #include <util/ConfigSource.h>
+
+#ifdef USE_RUY_GEMV
```

Comment:
`USE_RUY_GEMV` macro is defined to check whether or not to use ruy library for hybrid fully connected layer. This macro might be undefined even if ruy library is supported.

Ruy library is currently built on every targets, I think we can remove this macro check. Or it would be better to define a new macro which becomes true when ruy library is supported and use that macro here.

Reply:
Thanks for your information. How about `USE_RUY` as new macro?

Reply:
@ragmani `USE_RUY` looks good to me!

Reply:
Okay, I will do so.

Reply:
If we always build RUY when we build ONERT, I wonder if we need to use `#ifdef USE_RUY` . Could this line be removed?

Reply:
I think `onert` have to support `PermuteLayer` without using ruy. I use `#ifdef USE_RUY` to support it.